interop/update_binaries.sh

```diff
@@ -0,0 +1,33 @@
+set -e
```

Comment:
Bless you for this script 🙏 I was really not looking forward to updating these haha